azure-kusto-data/source/client.js

```diff
@@ -21,11 +21,12 @@ module.exports = class KustoClient {
 
 
     execute(db, query, callback, properties) {
-        if (query.startsWith(".")) {
-            return this.executeMgmt(db, query, callback, properties);
+        let trimmedQuery = query.trim();
```

Comment:
you can just reassign variable
```suggestion
        query = query.trim();
```